### ISSUE TITLE
SA1623 is triggered for private setters in private nested classes

### ISSUE THREAD

**Issue by 0x084E** [DotNetAnalyzers/StyleCopAnalyzers#1816]
I don't know if this caused by #1813 but I just mentioned that the following raises SA1623:

```csharp
public class Test
{
    private class NestedTest
    {
        /// <summary>
        /// Gets the test property.
        /// </summary>
        internal string TestProperty { get; private set; }
     }
}
```

**Comment by vweijsters** [comment]
#1813 has not been merged yet, so unless you are testing the pull request, the bug is in the master at the moment.

**Comment by 0x084E** [comment]
... sure, could have seen that. Than it might be vice versa and it will be fixed with #1813.

**Comment by sharwell** [comment]
Duplicate of #1811
